### PR TITLE
 Extract a utility method to get MySQL connection properties

### PR DESCRIPTION
**Pull request by ksobolew**
It will return a set of properties to be passed to JDBC, derived from the configuration.

### CODE REVIEW

File: plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java [hunk 1]
```diff
@@ -81,4 +82,21 @@ public MySqlConfig setDriverUseInformationSchema(boolean driverUseInformationSch
         this.driverUseInformationSchema = driverUseInformationSchema;
         return this;
     }
+
+    public Properties getConnectionProperties()
```
Comment (electrum) [inline plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java:86]: Config classes should only contain config, not other code. Make this a utility method in `MySqlClientModule` or another location.

**Review by electrum — CHANGES_REQUESTED** [review]

File: plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java (hunk 1 above)
Comment (ksobolew) [reply plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java]: Noted :) (And fixed)

**Comment by ksobolew** [comment]
@electrum PTAL

**Comment by ksobolew** [comment]
bump

**Review by kokosing — APPROVED** [review]

**Pull request merged**